tutorials/hello/hello_frontend/src/components/newAccount.js

```diff
@@ -0,0 +1,57 @@
+import { Container, Divider } from 'semantic-ui-react';
+import FixedMenuLayout from '../layout/header';
+import { passphrase, cryptography } from '@liskhq/lisk-client/browser';
+import React, { useState, useEffect } from "react";
+
+
+
+function NewAccount() {
+    const [accounts, createNewAccount] = useState('');
+
+    useEffect(() => {
+        createAccount()
+    }, [])
+
+    const createAccount = async () => {
+        const accountKeyPath = "m/44'/134'/" + Math.floor(Math.random() * 103) + "'";
```

Comment:
I think it is because of this line. It should always use the default value, not random.

Reply:
Yes, you're right. Code update.